Lib/concurrent/futures/process.py

```diff
@@ -855,3 +856,29 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def terminate_workers(self, signal=signal.SIGINT):
+        """Attempts to terminate the executor's workers using the given signal.
+        Iterates through all of the current processes and sends the given signal if
+        the process is still alive.
+
+        After terminating workers, the pool will be in a broken state and no longer usable.
+
+        Args:
+            signal: The signal to send to each worker process. Defaults to
+                signal.SIGINT.
+        """
+        if self._processes:
```

Comment:
- To reduce the indentation level, I suggest checking `if not self._processes: return` instead.
- You should also add some tests.

Reply:
Added some tests.